persistence-jdbc/core/src/main/scala/com/lightbend/lagom/internal/persistence/jdbc/SlickOffsetStore.scala

```diff
@@ -82,22 +82,24 @@ private[lagom] class SlickOffsetStore(system: ActorSystem, val slick: SlickProvi
   private val offsets = TableQuery[OffsetStore]
 
   private val startupTask = if (slick.autoCreateTables) {
-    implicit val timeout = Timeout(config.globalPrepareTimeout)
-    ClusterStartupTask(
-      system,
-      "slickOffsetStorePrepare",
-      createTables,
-      config.globalPrepareTimeout,
-      config.role,
-      config.minBackoff,
-      config.maxBackoff,
```

Comment:
did not understood how these line retries pardon my lack of knowldge.
we are also facing same error

Reply:
Facing what error? This re-executes the startup task each time prepare is called (by moving the `askExecute` into `runPreparations` rather than here).